chao_garden/src/init.c

```diff
@@ -0,0 +1,36 @@
+#include "global.h"
+#include "agb_flash.h"
+#include "m4a.h"
+
+extern IntrFunc gIntrTable[4];
+
+// TCG = Tiny Chao Garden
+struct TCG_SaveSectorHeader {
+    u32 security, version;
+};
+
+struct TCG_SaveSectorData {
+    struct TCG_SaveSectorHeader header;
+
+    u8 padding_8[0x850];
+}; /* size:0x858 according to sub_02000248 */
+
+// TCG = Tiny Chao Garden
+#define TCG_FIRST_SAVE_SECTOR 10
+#define TCG_SAVE_SECTOR_COUNT 6
+
+extern struct TCG_SaveSectorData gUnknown_03003B80;
+
+void sub_02000228(IntrFunc func)
```

Comment:
This is also still part of main.c, and likely `main.c` is actually `core.c` or something? Anyway this function is likely `SetVBlankIntr` https://github.com/freshollie/sa2/blob/main/chao_garden/asm/crt0.s#L39 as `gIntrTable[1]` is the VBlankIntr

Reply:
Oh yeah, you're right! I somehow couldn't find the name before.